src/main/java/com/gmail/yuramitryahin/model/User.java

```diff
@@ -6,7 +6,15 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
-
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
```suggestion
@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
```
why do you use this annotation?

Reply:
that don't write code for constructors, getters, setters, etc. Read about lombok, very useful case